pages/storage_and_backup/object_storage/Versioning_Doc/guide.en-gb.md

```diff
@@ -0,0 +1,129 @@
+---
+title: 'Getting Started with Versioning Using OVHcloud APIs'
+excerpt: 'Learn how to enable and manage versioning for your OVHcloud Object Storage buckets using APIs. Future updates will cover the OVHcloud Control Panel.'
+updated: 2024-06-26
+---
+
+## Objective
+
+**This guide explains how to enable and manage versioning for your OVHcloud Object Storage buckets using APIs.**
+
+## Requirements
+
+- A Public Cloud project in your OVHcloud account
```

Comment:
The Requirements list must be linked to other guides, just like what is done on other similar guides.